authentication/copilot/backend/manifest.yml

```diff
@@ -36,7 +36,7 @@ network:
 # Optional fields for more advanced use-cases.
 #
 variables:                    # Pass environment variables as key value pairs.
-  CLIENT_ID: f67916ce-de33-4e2f-a8e3-cbd5f6459c30
+  OAUTH_CLIENT_ID: f67916ce-de33-4e2f-a8e3-cbd5f6459c30
```

Comment:
Hardcoded ID needs removing

Reply:
I've left the values in our deployment files as they seem to be in the correct place there (values specific to our deployment) and I don't think we have a reason to treat it as confidential information.